### ISSUE TITLE
ability to use html code in footer

### ISSUE THREAD

**Issue by heenamahour** [d3plus/d3plus#391]
It will be great if one can use html within footer() as well apart from setting value and link

**Comment by itsmejainh** [comment]
yeah.. nice to have feature..+1 
Just like we can specify html in tootlip()

**Comment by davelandry** [comment]
agreed, would be great, but it would take a little bit of restructuring as the footer is currently an SVG element (while the tooltip is an html DIV)

**Comment by heenamahour** [comment]
hope it gets resolved soon, @davelandry  is there a quick hack to try it meanwhile?

**Comment by davelandry** [comment]
The only way around it at the moment would be to make your own <div> element and position it at the bottom of the visualization.
